src/main/java/pl/kinga/leczenie/uzytkownik/Uzytkownik.java

```diff
@@ -0,0 +1,81 @@
+package pl.kinga.leczenie.uzytkownik;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDate;
+
+@Entity
+public class Uzytkownik {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String imie;
+    private String nazwisko;
+    @DateTimeFormat(pattern = "yyyy-MM-dd")
+    private LocalDate data_urodzenia;
```

Comment:
Zmienne w javie nazywamy camelCasem - czyli powinno być dataUrodzenia, w bazie danych to z automatu zostanie zmienion na data_urodzenia